functions.md

```diff
@@ -826,10 +826,13 @@ format-time(null, "yyyy-MM-dd")                  => null
 
 ## Miscellaneous functions
 
-### _parse-url(url) -> object_
+### _parse-url(url, throwOnFailure?) -> object_
 
 Parses `url` and returns an object with keys [`scheme`, `userinfo`, `host`, `port` `path`, `query`, `parameters`, `fragment` ]
 
+If the optional `throwOnFailure` argument is not specified invalid URLs will generate an exception. If `throwOnFailure` is set to `false` the `parse-url` returns
+an object indicating an error occurred.
+
```

Comment:
We need to document the shape of that object, and have an example.

Reply:
I added some examples below but I agree we could add more documentation on the shape both when good and bad input

Reply:
Updated with more detaild examples.